sycl/test-e2e/DeviceImageBackendContent/basic_test.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %{build} -std=c++20 -o %t.out
```

Comment:
We had some complains from Codeplay developers about such tests.

Reply:
Can you point me to a thread where I can have a look at the concerns?

Reply:
I don't have a link, try searching through git log with changes containing `-std=++20`